plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/project/TestProjectCreator.java

```diff
@@ -90,7 +90,7 @@ protected void after() {
       try {
         project.delete(true, null);
       } catch (CoreException ex) {
-        fail("Could not delete project");
+        fail("Could not delete project: " + ex.getMessage());
```

Comment:
Suggestion to replace these with:

    throw new AssertionError("Could not delete project", ex);

That should show the sub-exception too.